test/e2e/flowaggregator_test.go

```diff
@@ -696,10 +698,15 @@ func checkPodAndNodeData(t *testing.T, record, srcPod, srcNode, dstPod, dstNode
 	assert.Contains(record, fmt.Sprintf("sourceNodeName: %s", srcNode), "Record does not have correct sourceNodeName")
 	// For Pod-To-External flow type, we send traffic to an external address,
 	// so we skip the verification of destination Pod info.
+	// Also, source Pod labels are different for Pod-To-External flow test.
```

Comment:
Previously, we are not checking anything for Pod-To-External flows. Now we are checking source pod labels in that case. I feel the comment is not clear. Please revise it.